src/pflask.c

```diff
@@ -44,6 +44,11 @@
 #include <sys/stat.h>
 #include <sys/wait.h>
 
+#if HAVE_LIBCAP_NG
+#include <cap-ng.h>
```

Comment:
I've made ``capabilities.c`` (in ``wscript``) built only when libcap-ng is available, hence the conditional ``capabilities.h`` include. But yes, I've removed the cap-ng.h include